packages/block-editor/src/hooks/test/supports.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * Internal dependencies
+ */
+import {
+	hasBlockClassNameSupport,
+	hasVariationClassNameSupport,
+} from '../supports';
+
+describe( 'hasBlockClassNameSupport', () => {
+	const blockName = 'block/name';
+
+	it( 'should default to true', () => {
+		const block = {
+			name: blockName,
+		};
+		expect( hasBlockClassNameSupport( block ) ).toEqual( true );
```

Comment:
Nitpick: when testing against boolean value the `===` strict equality operator is more appropriate:
```suggestion
		expect( hasBlockClassNameSupport( block ) ).toBe( true );
```
It applies to all test cases added.

Reply:
Ah, good spot! I'll change those 👍

Reply:
9b5f0b60f3a98ee5129b701f76ab8218a572b7fe and 611dd128dac